goss-testing/suites/ncn-healthcheck-master.yaml

```diff
@@ -34,6 +34,7 @@ gossfile:
   ../tests/goss-default-gateway-points-to-CMN-gateway.yaml: {}
   ../tests/goss-dns-nodes.yaml: {}
   ../tests/goss-interface-has-ip.yaml: {}
+  ../tests/goss-bss-basecamp-valid.yaml: {}
```

Comment:
We don't want to add it to this suite. It's already in the corresponding -single.yaml suite, so that it will only run once.
```suggestion
```